### PR TITLE
Trial json viewer developed by vanilla JS

### PR DESCRIPTION
**Pull request by victoria34**
screen shot 2018-07-03 at 6 01 48 pm

### CODE REVIEW

File: data-example/NCT001.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "long_title" : "A Phase 1 Multiple Ascending Dose Study of DS-3032b, an Oral MDM2 Inhibitor, in Subjects With Advanced Solid Tumors or Lymphomas",
```
Comment (zhx828) [inline data-example/NCT001.json:2]: Are these mock data?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
-# trial-viewer
-A pure vanilla json viewer for trials
+# TrialViewer
+TrialViewer is a vanilla JavaScript tool for visualising trial JSON data. It is inspired from [JSONViewer].
```
Comment (zhx828) [inline README.md]: Did you change the JSONViewer library?

Comment (victoria34) [reply README.md]: No. I just use it directly.

File: data-example/NCT001.json (hunk 1 above)
Comment (victoria34) [reply data-example/NCT001.json:2]: This trial is got from public curation platform. I changed nct_id.

File: src/json-viewer.js [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+            callback(xhr.responseText);
+        }
+    };
+    xhr.send(null);
+}
+
+/**
+ * JSONViewer - by Roman Makudera 2016 (c) MIT licence.
+ */
+JSONViewer = (function() {
```
Comment (zhx828) [inline src/json-viewer.js]: I think at the end of the day, this file will be used separately. Line 1-21 is just giving example how to use this lib, so we probably should just separate it to a lib.

File: style/json-viewer.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
+.json-viewer {
```
Comment (zhx828) [inline style/json-viewer.css:1]: Are these styles coming from the lib too?

File: src/json-viewer.js (hunk 3 above)
Comment (victoria34) [reply src/json-viewer.js]: Ok, I'll separate the file.

File: README.md (hunk 2 above)
Comment (zhx828) [reply README.md]: Then it is not inspired. We just created a repo to demo how to use this lib.

File: style/json-viewer.css (hunk 4 above)
Comment (victoria34) [reply style/json-viewer.css:1]: Yes

File: data-example/NCT001.json (hunk 1 above)
Comment (zhx828) [reply data-example/NCT001.json:2]: Sure

**Review by zhx828 — APPROVED** [review]

**Pull request merged**